app-admin/prelude-manager/prelude-manager-3.0.0.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools eutils systemd
+
+DESCRIPTION="Bus communication for all Prelude modules"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${PF}.tar.gz"
```

Comment:
Your use of `${PF}` in SRC_URI may not result in what you expect as it may include the Gentoo-specific revision too. See: https://devmanual.gentoo.org/ebuild-writing/variables/index.html

I believe the correct SRC_URI should be:
```
SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${P}.tar.gz"
```

Ditto other ebuilds